modules/retrieving-connectivity-mapping-information-from-a-kubernetes-manifest-directory.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * operating/build-time-network-policy-tools.adoc
+
+:_content-type: PROCEDURE
+[id="retrieving-connectivity-mapping-information-from-a-kubernetes-manifest-directory_{context}"]
+= Retrieving connectivity mapping information from a Kubernetes manifest directory
+
+In the following example, link:https://github.com/np-guard/netpol-analyzer/tree/main/tests/netpol-analysis-example-minimal[`netpol-analysis-example-minimal/`] is the directory path containing the Kubernetes workloads and network policy manifests to be analyzed.
+
+.Procedure
+
+* Run the following command to retrieve the connectivity mapping information from link:https://github.com/np-guard/netpol-analyzer/tree/main/tests/netpol-analysis-example-minimal[`netpol-analysis-example-minimal/`]:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl netpol connectivity map netpols-analysis-example-minimal/
+----
++
+The following is an example output generated by the `roxctl netpol connectivity map` command in `md` format:
+
+** link:https://github.com/np-guard/netpol-analyzer/blob/main/tests/netpol-analysis-example-minimal/connlist_output.md[Connectivity mapping information]
+
+The output shows you a table with a list of allowed connectivity lines. Each connectivity line consists of three parts: source (`src`), destination (`dst`), and allowed connectivity attributes (`conn`).
+
+You can interpret `src` as the source endpoint, `dst` as the destination endpoint, and `conn` as the allowable connectivity attributes. You have the option to designate either the source or the destination as one of the analyzed cluster workloads, or to specify an IP address range. A typical cluster workload has the format `namespace/name[Kind]`, for example, `default/backend[Deployment]`.
```

Comment:
Delete the sentence "You have the option to designate either the source or the destination as one of the analyzed cluster workloads, or to specify an IP address range."